cmd/openfga/openfga.go

```diff
@@ -63,6 +67,14 @@ type svcConfig struct {
 	// RequestTimeout is a limit on the time a request may take. If the value is 0, then there is no timeout.
 	RequestTimeout time.Duration `default:"0s" split_words:"true"`
 
+	GRPCTLSEnabled  bool   `default:"false" envconfig:"GRPC_TLS_ENABLED"`
+	GRPCTLSCertPath string `envconfig:"GRPC_TLS_CERT_PATH"`
+	GRPCTLSKeyPath  string `envconfig:"GRPC_TLS_KEY_PATH"`
+
+	HTTPGatewayTLSEnabled  bool   `default:"false" split_words:"true"`
```

Comment:
Any reason why the config leaks the usage of `grpc-gateway`?

Reply:
Drop the `Gateway` I guess?